tests/CppUTest/UtestTest.cpp

```diff
@@ -218,12 +222,26 @@ TEST(UtestShell, TestStopsAfterSetupFailure)
 }
 
 #if CPPUTEST_USE_STD_CPP_LIB
-static void thrownUnknownExceptionMethod_()
+#ifdef NEEDS_DISABLE_UNREACHABLE_CODE
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wunreachable-code"
+#endif /* NEEDS_DISABLE_UNREACHABLE_CODE */
+_no_return_ static void thrownUnknownExceptionMethod_()
 {
     throw 33;
```

Comment:
if (shouldAlwaysThrow) throw 33; ?

Reply:
A simple *if* condition caused an "always-true" warning, but a little more trickery solved it.